src/Sarif/Autogenerated/SarifRewritingVisitor.cs

```diff
@@ -805,6 +813,13 @@ public virtual Run VisitRun(Run node)
                 }
 
                 node.ExternalPropertyFiles = VisitNullChecked(node.ExternalPropertyFiles);
+                if (node.ThreadFlowLocations != null)
+                {
+                    for (int index_0 = 0; index_0 < node.ThreadFlowLocations.Count; ++index_0)
+                    {
+                        node.ThreadFlowLocations[index_0] = VisitNullChecked(node.ThreadFlowLocations[index_0]);
+                    }
+                }
```

Comment:
same comment as above: this is strictly not part of this PR, but when i regenerated this file i noticed this additional check and thought of bringing is to attention in case it was mistakenly not included earlier?

Reply:
yes! this was a bug in the SDK! the visitor wouldn't visit newly added things.

---
In reply to: [262333747](https://github.com/Microsoft/sarif-sdk/pull/1328#discussion_r262333747) [](ancestors = 262333747)